index.html

```diff
@@ -19,10 +19,11 @@
             <a class="logo">Femi</a>
             <a class="togglebar" href="#"><img src="Images/Icon-Menu.png" alt="menu icon"></a>
             <ul class="nav-items">
-                <li><a href="#">portfolio</a></li>
-                <li><a href="#">about</a></li>
-                <li><a href="#">contact</a></li>
-                <li><a href="#"><img src="Images/Icon-Mail.png" alt="mail icon"></a></li>
+                <li class="close"><a href="#"><img src="Images/Icon-Cancel.png" alt="Cancel Icon"></a></li>
```

Comment:
- [ ] This **X**  should not be present in desktop version, please remove it ![Screenshot from 2021-10-04 20-21-18](https://user-images.githubusercontent.com/62705947/135903720-6db753a9-4a6e-4b00-956d-d50292f564b2.png)
 - [ ]